src/app/lib/utilities/srcSet/index.test.js

```diff
@@ -80,6 +80,21 @@ describe('create srcset', () => {
       summary:
         'should return empty srcsets and null for mime types if the srcsets are malformed',
     },
+    {
+      originCode: 'cpsdevpb',
+      locator: 'testland',
+      width: 2048,
+      expected: {
+        primarySrcset:
+          'https://ichef.bbci.co.uk/ace/ws/240/cpsdevpb/testland 240w, https://ichef.bbci.co.uk/ace/ws/320/cpsdevpb/testland 320w, https://ichef.bbci.co.uk/ace/ws/480/cpsdevpb/testland 480w, https://ichef.bbci.co.uk/ace/ws/624/cpsdevpb/testland 624w, https://ichef.bbci.co.uk/ace/ws/800/cpsdevpb/testland 800w',
+        primaryMimeType: null,
+        fallbackSrcset:
+          'https://ichef.bbci.co.uk/ace/ws/240/cpsdevpb/testland 240w, https://ichef.bbci.co.uk/ace/ws/320/cpsdevpb/testland 320w, https://ichef.bbci.co.uk/ace/ws/480/cpsdevpb/testland 480w, https://ichef.bbci.co.uk/ace/ws/624/cpsdevpb/testland 624w, https://ichef.bbci.co.uk/ace/ws/800/cpsdevpb/testland 800w',
+        fallbackMimeType: null,
+      },
+      summary:
+        'should return no file extension when none on locator and no MIME types', // do we need this test to cover null mime types?
+    },
```

Comment:
Is this a real world scenario?

Reply:
I don't knoe but as I un-nulled the mime types in the other tests I added a new test with them as null just in case that was important for some reason. I don't know if it represents something real or if the mime types were just null because they weren't originally important in the test.

Reply:
I wonder if we should remove a test if we aren't clear what the real world scenario is?